client/components/section-nav/tabs.jsx

```diff
@@ -144,12 +143,10 @@ class NavTabs extends Component {
 	}
 
 	getTabWidths() {
-		let totalWidth = 0;
-
-		this.tabRefMap.forEach( ( tabElement ) => {
-			const tabWidth = ReactDom.findDOMNode( tabElement ).offsetWidth;
-			totalWidth += tabWidth;
-		} );
+		const totalWidth = Array.from( this.tabRefMap.values() ).reduce(
+			( sum, tabElement ) => sum + tabElement.offsetWidth,
+			0
+		);
```

Comment:
fixes this:
> Understand this error
tabs.jsx:239 Warning: findDOMNode is deprecated and will be removed in the next major release. Instead, add a ref directly to the element you want to reference. Learn more about using refs safely here: https://reactjs.org/link/strict-mode-find-node

Reply:
Extracted these changes into #104844